snuba/datasets/storages/errors_v2.py

```diff
@@ -0,0 +1,56 @@
+from snuba import util
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.errors_processor import ErrorsProcessor
+from snuba.datasets.errors_replacer import ErrorsReplacer, ReplacerState
+from snuba.datasets.message_filters import KafkaHeaderFilter
+from snuba.datasets.schemas.tables import WritableTableSchema
+from snuba.datasets.storage import WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.errors_common import (
+    all_columns,
+    mandatory_conditions,
+    promoted_tag_columns,
+    query_processors,
+    query_splitters,
+    required_columns,
+)
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+from snuba.query.processors.conditions_enforcer import ProjectIdEnforcer
+from snuba.subscriptions.utils import SchedulingWatermarkMode
+from snuba.utils.streams.topics import Topic
+
+schema = WritableTableSchema(
+    columns=all_columns,
+    local_table_name="errors_local",
+    dist_table_name="errors_dist",
+    storage_set_key=StorageSetKey.ERRORS_V2,
+    mandatory_conditions=mandatory_conditions,
+    part_format=[util.PartSegment.RETENTION_DAYS, util.PartSegment.DATE],
+)
+
+storage = WritableTableStorage(
+    storage_key=StorageKey.ERRORS_V2,
+    storage_set_key=StorageSetKey.ERRORS_V2,
```

Comment:
This file is similar to https://github.com/getsentry/snuba/blob/master/snuba/datasets/storages/errors.py except for the storage keys over here and over here https://github.com/getsentry/snuba/pull/2349/files#diff-85d48fb2f4bfd3f7b16a317b13b37953c66c95ef61ff41544af53e23536f8178R26